dev

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+yarn
+env FORCE_COLOR=0 \
```

Comment:
Why this?

Reply:
`FORCE_COLOR=0` tells Parcel not to rewrite lines in the terminal, which makes for more readable output.